### PR TITLE
Build Simple Calculator with Python GUI

### PR DESCRIPTION
**Pull request by Geoveza**
This is only as a trial phase for python gui based calculators

### CODE REVIEW

File: JavaScript/profanity_checker.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+def profanity_checker(text) :
+    connection = urllib.urlopen("http://www.wdylike.appspot.com/?q="+text)
+    output = connection.read()
+    
+    if output == "true":
+        print("\nProfanity Alert !! for \'"+text+"\'")
+    elif output == "false":
+        print("\nNo Cuss Words :) in \'"+text+"\'")
+        
+read_text()
```
Comment (moeez96) [inline JavaScript/profanity_checker.py]: Please add eof line

File: Python/simple-calculator.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+b3.grid(column=7,row=10)
+
+b4=tk.Button(gui,text='/',command=abf,bg='pink',activebackground='orange')
+b4.grid(column=8,row=10)
+
+e1=tk.Entry(gui)
+e1.grid(column=2,row=2)
+
+e2=tk.Entry(gui)
+e2.grid(column=3,row=3)
```
Comment (moeez96) [inline Python/simple-calculator.py]: Please add eof line

File: JAVA/threadsRunnable.java [hunk 3]
```diff
@@ -1,11 +1,14 @@
 import java.io.*;
-public class nsp{
-	public static void main(String[]args){
-		Thread t=new Thread(){
-			public void run(){
-				System.out.println("java is ");
-			}
-		};
-			t.start();	
+class demo implements Runnable{
```
Comment (Pr0methean) [inline JAVA/threadsRunnable.java]: By Java conventions, class names should be in UpperCamelCase, and the filename should match the class name.

File: Python/simple-calculator.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
+import tkinter as tk
+gui=tk.Tk()
+gui.title('SIMPLE_CALCULATOR')
+gui.geometry('400x400')
+l1=tk.Label(gui,text='ans')
+l1.grid(column=0,row=0)
+
+def abc():
```
Comment (Pr0methean) [inline Python/simple-calculator.py]: Use more descriptive names like "add", "subtract"...

File: Python/simple-calculator.py [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    result=int(a)*int(b)
+    l1.configure(text=result)
+
+
+def abf():
+    a=e1.get()
+    l1.configure(text=a)
+    b=e2.get()
+    l1.configure(text=b)
+    result=int(a)/int(b)
```
Comment (Pr0methean) [inline Python/simple-calculator.py]: Handle division by zero?

**Pull request merged**